packages/runtime-js/build.gradle.kts

```diff
@@ -11,92 +11,41 @@
  * License for the specific language governing permissions and limitations under the License.
  */
 
-@file:Suppress(
-  "UnstableApiUsage",
-  "unused",
-  "DSL_SCOPE_VIOLATION",
-)
-
-import ElidePackages.elidePackage
+import elide.internal.conventions.elide
+import elide.internal.conventions.kotlin.KotlinTarget
+import elide.internal.conventions.publishing.publish
 
 plugins {
-  id("dev.elide.build.native.lib")
-}
-
-group = "dev.elide"
-version = rootProject.version as String
-
-elidePackage(
-  id = "runtime-js",
-  name = "Elide JavaScript Runtime",
-  description = "Package providing the JavaScript plugin for the Elide runtime.",
-)
-
-kotlin {
-  explicitApi()
+  kotlin("jvm")
+  id("elide.internal.conventions")
 }
 
-val initializeAtBuildTime = listOf(
-  "kotlin.DeprecationLevel",
-  "kotlin.annotation.AnnotationRetention",
-  "kotlin.annotation.AnnotationTarget",
-  "kotlin.coroutines.intrinsics.CoroutineSingletons",
-)
-
-val initializeAtBuildTimeTest = listOf(
-  "org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter",
-  "org.junit.platform.launcher.core.LauncherConfig",
-)
-
-val sharedLibArgs = listOf(
-  "-H:+AuxiliaryEngineCache",
-)
```

Comment:
are these making it in elsewhere?

Reply:
looks like native-image configuration was lost in translation for these projects (since they are not really being published as shared native libraries I didn't pick that up during the migration)

I'm restoring these minus a few shared lines that are handled by the conventions.

Reply:
update: I apparently mixed up which project this was on. For the `runtime-*` family, the original buildscript was based on the one for `graalvm`, which includes native image configuration. Initially I just copied that script and reused for those projects, but as far as I know those native image components are not being used or published, which is why that configuration was removed.

Reply:
yes just good to keep in mind that we will probably need to:

1) inject custom native-image args or a given lib
2) use baseline flags which apply to all native libs

we don't need to implement this now, but at least preserve the previous behavior and we're good